### DISCUSSION TITLE
Add CustomAttr function

### DISCUSSION THREAD

**Discussion by Lord-Y** [maragudk/gomponents#158] [Ideas] [answered]
Hello @markuswustenberg Thank you for creating `gomponents`. This is such a clean and easy way to generate HTML code.
So far everything have been pretty good for me.
The only thing I think is missing is a custom attribute func like:
```go
func CustomAttr(k, v string) g.Node {
	return g.Attr(k, v)
}
```
It's very useful when you what to add [hreflang](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/link#hreflang) attribute or any other attribute that is not currently supported.

What do you think?

**Comment by markuswustenberg** [comment, marked as answer]
Hi @Lord-Y. Just use g.Attr directly and you have your custom attribute. 😊

**Reply by Lord-Y** [reply]
That's true 🤣

**Reply by markuswustenberg** [reply]
That said, if there are attributes missing that you think should be included (standard HTML 5, not deprecated, browser-specific, or experimental), feel free to file an issue or make a PR! 😊
